spec/features/create_audio_object_spec.rb

```diff
@@ -0,0 +1,156 @@
+
+    # create manifest.csv file and scp it to preassembly staging directory
+    File.write(local_manifest_location, preassembly_manifest_csv)
+    `scp #{local_manifest_location} #{remote_manifest_location}`
+    raise("unable to scp #{local_manifest_location} to #{remote_manifest_location} - got #{$CHILD_STATUS.inspect}") unless $CHILD_STATUS.success?
+
+    # create media_manifest.csv and scp it to preassembly staging directory
+    File.write(local_media_manifest_location, media_manifest_csv)
+    `scp #{local_media_manifest_location} #{remote_manifest_location}`
+    raise("unable to scp #{local_media_manifest_location} to #{remote_manifest_location} - got #{$CHILD_STATUS.inspect}") unless $CHILD_STATUS.success?
+
+    visit 'https://sul-preassembly-stage.stanford.edu/'
+    expect(page).to have_selector('h3', text: 'Complete the form below')
+
+    fill_in 'Project name', with: preassembly_project_name
+    select 'Pre Assembly Run', from: 'Job type'
+    select 'Media', from: 'Content structure'
+    fill_in 'Bundle dir', with: preassembly_bundle_dir
+    select 'Media', from: 'Content metadata creation'
+
+    click_button 'Submit'
+    exp_str = 'Success! Your job is queued. A link to job output will be emailed to you upon completion.'
+    expect(page).to have_content exp_str
+
+    # go to job details page, download result
+    first('td  > a').click
+    expect(page).to have_content preassembly_project_name
+    # wait for preassembly background job to finish
+    Timeout.timeout(100) do
+      loop do
+        page.evaluate_script('window.location.reload()')
+        break if page.has_link?('Download')
+      end
+    end
+    click_link 'Download'
+    wait_for_download
+    yaml = YAML.load_file(download)
+    expect(yaml[:status]).to eq 'success'
+
+    visit "https://argo-stage.stanford.edu/view/#{audio_druid}"
+
+    # Wait for accessioningWF to finish
+    Timeout.timeout(100) do
+      loop do
+        page.evaluate_script('window.location.reload()')
+        break if page.has_text?('v1 Accessioned')
+      end
+    end
+
+    # ensure files are all there, organized into specified resources
+    expect(page).to have_selector('.blacklight-content_type_ssim', text: 'media')
+    expect(page).to have_selector('#document-contents-section > .resource-list > li.resource', text: 'Resource (1) audio')
+    expect(page).to have_selector('#document-contents-section > .resource-list > li', text: 'Audio file 1')
+    expect(page).to have_selector('#document-contents-section > .resource-list > li.resource', text: 'Resource (2) text')
+    expect(page).to have_selector('#document-contents-section > .resource-list > li', text: 'Transcript')
+    audio_files.each do |af|
+      fname = druid_prefixed_name(af, audio_druid)
+      expect(page).to have_text(fname)
+    end
+  end
+end
+
+def druid_prefixed_name(old_filename, druid)
```

Comment:
I'm up in the air on whether this method is a useful addition or more noise than it's worth. `"#{druid}_#{old_filename}"` is shorter than `druid_prefixed_name(old_filename, druid)` and it just as clear re: intent, I think? What do you think?

Reply:
excellent point.  Will do.